lib/membrane_ffmpeg_swscale/scaler/scaler.ex

```diff
@@ -33,33 +33,33 @@ defmodule Membrane.FFmpeg.SWScale.Scaler do
               ],
               use_shm?: [
                 type: :boolean,
-                desciption:
+                description:
                   "If true, native scaler will use shared memory (via `t:Shmex.t/0`) for storing frames",
                 default: false
               ]
 
   def_input_pad :input,
     demand_unit: :buffers,
     demand_mode: :auto,
-    caps: {RawVideo, pixel_format: :I420, aligned: true}
+    accepted_format: %RawVideo{pixel_format: :I420, aligned: true}
 
   def_output_pad :output,
     demand_mode: :auto,
-    caps: {RawVideo, pixel_format: :I420, aligned: true}
+    accepted_format: %RawVideo{pixel_format: :I420, aligned: true}
 
   @impl true
-  def handle_init(options) do
+  def handle_init(_ctx, options) do
     state =
       options
       |> Map.from_struct()
       |> Map.put(:native_state, nil)
 
-    {:ok, state}
+    {[], state}
   end
 
   @impl true
   def handle_process(:input, _buffer, _context, %{native_state: nil} = _state) do
-    raise(RuntimeError, "uninitialized state: Scaler did not receive caps")
+    raise(RuntimeError, "uninitialized state: Scaler did not receive stream_format")
```

Comment:
Since Core 0.10, the caps (now stream format) are enforced and this can never happen - you can remove the clause of `handle_process`